lutris/game.py

```diff
@@ -502,12 +503,16 @@ def stop(self):
         if self.game_thread:
             jobs.AsyncCall(self.game_thread.stop, None, killall=self.runner.killall_on_exit())
         self.state = self.STATE_STOPPED
+        if not self.timer.finished:
+            self.timer.end()
+            self.playtime = self.timer.duration / 3600 + self.playtime
 
     def on_game_quit(self):
         """Restore some settings and cleanup after game quit."""
 
-        self.timer.end_t()
-        self.playtime = self.timer.increment(self.playtime)
+        if not self.timer.finished:
+            self.timer.end()
+            self.playtime = self.timer.duration / 3600 + self.playtime
```

Comment:
```self.playtime += self.timer.duration / 3600```

conversion should be done within the Timer class depending on what format has been set when instantiating.

Reply:
Ok, thought its not necessary since its in seconds but I guess you'll right I'll add that

Reply:
I used a simplistic conversation method, is that good enough?